tests/mhlo-fusion.mlir

```diff
@@ -95,3 +95,15 @@ func @reduce_2(%arg0: tensor<?x?xf32>, %arg1: tensor<?x?xf32>) -> (tensor<?x?xf3
   %4 = "mhlo.add"(%3, %3) : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
   return %1, %4 : tensor<?x?xf32>, tensor<?xf32>
 }
+
+// -----
+
+// CHECK-LABEL: func @fused_unfused_mix
```

Comment:
I'd add a CHECK for ensuring the fusion is actually produced, and a comment to explain what we're testing:

```
// This case use to fail the verifier by breaking dominance, we
// just verify that we generate a fusion and don't crash.
// CHECK: mhlo.fusion
```

Reply:
thanks